components/progress/Steps.tsx

```diff
@@ -14,14 +15,15 @@ const Steps: React.FC<ProgressStepsProps> = (props) => {
   const {
     size,
     steps,
+    rounding,
```

Comment:
Do you think this would be better?
```ts
const {
  rounding: customRounding = Math.round,
  ...
} = props;

const current = customRounding(steps * (percent / 100));
```